src/main/java/com/eternalcode/parcellockers/parcel/command/ParcelCommand.java

```diff
@@ -22,7 +24,7 @@
 import java.util.Objects;
 import java.util.UUID;
 
-@Route(name = "parcel")
+@Command(name = "parcel", aliases = { "parcellockers" })
```

Comment:
```suggestion
@Command(name = "parcel")
```
Konfliktuje z [ParcelLockersCommand.java](https://github.com/EternalCodeTeam/ParcelLockers/pull/89/files#diff-834a2a970bb59c224aeff1b5e5de1ff52e8b071691bcbd25e49f98cbec336c14)